CentOS.repo

```diff
@@ -0,0 +1,15 @@
+[AppStream]
```

Comment:
Mixing up the CentOS packages with RHEL may cause unexpected behavior. I’d suggest removing those CentOS repos and stick to RHEL repos as long as we are building on RHEL nodes, like PSI.